control_panel_api/tests/test_models.py

```diff
@@ -192,13 +192,18 @@ def test_arn(self):
 
     @patch('control_panel_api.services.create_bucket')
     def test_bucket_create(self, mock_create_bucket):
+        url = 'http://foo.com/'
```

Comment:
No need to change this unless you want to, but it's always best to use `example.com` for test domains. `foo.com` is a real domain.

Reply:
Good point - bad habit.